src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -792,22 +793,53 @@ function toParameterFormat(str) {
     return str.replace(/\s/g, '+');
 }
 
-function bindCopyButton() {
-    $('#button_copy').on('click', function(e) {
-        e.preventDefault();
+/**
+ * Adds event handler to load 'copy question' modal contents by ajax.
+ */
+function setupQuestionCopyModal() {
+    $('#copyModal').on('show.bs.modal', function(event) {
+        var button = $(event.relatedTarget); // Button that triggered the modal
+        var actionlink = button.data('actionlink');
+        var courseid = button.data('courseid');
+        var fsname = button.data('fsname');
         
-        var questionRows = $('#copyTableModal >tbody>tr');
-        if (questionRows.length) {
-            setStatusMessage('', StatusType.WARNING);
-            $('#copyModal').modal('show');
-        } else {
-            setStatusMessage(FEEDBACK_QUESTION_COPY_INVALID, StatusType.DANGER);
-        }
-       
-        return false;
+        var $questionCopyStatusMessage = $('#question-copy-modal-status');
+        var $copyTableModal = $('#copyTableModal');
+        $.ajax({
+            type: 'GET',
+            url: actionlink + '&courseid=' + encodeURIComponent(courseid)
+                            + '&fsname=' + encodeURIComponent(fsname),
+            beforeSend: function() {
+                $('#button_copy_submit').prop('disabled', true);
+                $copyTableModal.html('Loading possible questions to copy. Please wait ...<br>'
+                                     + "<img class='margin-center-horizontal' src='/images/ajax-loader.gif'/>");
+                $questionCopyStatusMessage.removeClass('alert alert-danger');
+            },
+            error: function() {
+                $copyTableModal.html('');
+                $questionCopyStatusMessage.html('Error retrieving questions. '
+                                                + 'Please close the dialog window and try again.');
+                $questionCopyStatusMessage.addClass('alert alert-danger');
+            },
+            success: function(data) {
+                $copyTableModal.replaceWith($(data).find('#copyTableModal'));
+                
+                var $questionRows = $('#copyTableModal > tbody > tr');
+                if ($questionRows.length) {
+                    $questionCopyStatusMessage.html('');
+                } else {
+                    $('#copyTableModal').html('');
+                    $questionCopyStatusMessage.addClass('alert alert-danger');
+                    $questionCopyStatusMessage.prepend('<br>').html(FEEDBACK_QUESTION_COPY_INVALID);
+                }
+            }
+        });
     });
+}
+
+function bindCopyButton() {
 
-    $('#button_copy_submit').on('click', function(e) {
+    $('body').on('click', '#button_copy_submit', function(e) {
```

Comment:
Is there any reason why it is done this way? Could we use `$('#button_copy_submit').click(function(e) {` instead, as the button is not a dynamically added element.